src/main/java/core/selenium/Browser.java

```diff
@@ -1,12 +1,12 @@
-package gui;
+package core.selenium;
 
 import io.github.bonigarcia.wdm.WebDriverManager;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
-public class GuiSetting {
+public class Browser {
 
     /**
      * Sets Browser type.
```

Comment:
maybe it will better if you replace this setBrowser for a factory

Reply:
solved